purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -38,6 +40,9 @@ open class PurchasesConfiguration(builder: Builder) {
         @set:JvmSynthetic @get:JvmSynthetic
         internal var observerMode: Boolean = false
 
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var showDeclinedPaymentMessagesAutomatically: Boolean = false
```

Comment:
Should we make this default to `true` already?

Reply:
Yeah, will just make it default to true. We shouldn't release tomorrow's automatic release though with this.

Reply:
Changed in https://github.com/RevenueCat/purchases-android/pull/1290/commits/31570b6fee3bcb3498d213352c1c45ef5d83dac2